central/policy/datastore/singleton.go

```diff
@@ -31,11 +31,12 @@ func initialize() {
 	if features.PostgresDatastore.Enabled() {
 		storage = policyPostgres.New(globaldb.GetPostgres())
 		indexer = policyPostgres.NewIndexer(globaldb.GetPostgres())
-		addDefaults(storage)
 	} else {
 		storage = boltdb.New(globaldb.GetGlobalDB())
 		indexer = index.New(globalindex.GetGlobalTmpIndex())
 	}
+
+	addDefaults(storage)
```

Comment:
Why is this moved out of the postgres if block?

Reply:
Related to point 9. in the description: "Fixed a bug in populating default policies in rocksdb and in postgres on startup.
". Basically fixed a bug since default policies must be added regardless of postgres or rockdb

Reply:
This is in the bolt store currently so now its duplicated

Reply:
sorry @theencee I meant bolt. I need to add defaults regardless of the store.

Reply:
That's correct. The comment on the addDefaults function remarks that the migration will need to take care of the fact we removed the removed_default_policies bucket so the migration will need to account for it